src/rgw/driver/rados/rgw_data_sync.cc

```diff
@@ -2405,7 +2405,9 @@ class RGWDataSyncShardNotifyCR : public RGWCoroutine {
         yield call(sync_env->bid_manager->notify_cr());
         if (retcode < 0) {
           tn->log(5, SSTR("ERROR: failed to notify bidding information" << retcode));
-          return set_cr_error(retcode);
+          if (retcode != -ETIMEDOUT) {
```

Comment:
we might just want to log and ignore _all_ error codes here, in case there are others that can show up